src/main/java/com/mhcure/logmerge/filereader/MhFileReader.java

```diff
@@ -103,7 +103,7 @@ public Map<Long, String> readFileUsingBufferedReader(String fileName) throws Exc
         boolean isEncryptedFile = false;
         Cipher cipherObject = null;
         if (!logFile.isFile()) {
-            System.out.println(fileName + MhMessagePropertiesFileReader.getMessage(MhMessageKeyEnum.IGNORE_INVALID_FILE.getKey()));
+            MhFileAggregatorHelper.printToConsole(fileName + MhMessagePropertiesFileReader.getMessage(MhMessageKeyEnum.IGNORE_INVALID_FILE.getKey()));
             return fileContentsMap;
         }
         logFileReader = new FileReader(logFile);
```

Comment:
line no 171 no need of variable declaration
final String ALGO = MhFileConstants.ALGO
u can directly use that constants

Reply:
I changed according to the comment